main.go

```diff
@@ -117,7 +117,15 @@ func main() {
 	router.GET("/api/whoami", func(ctx *gin.Context) {
 		user := getCurrentUser(ctx)
 		if user != nil {
-			ctx.JSON(http.StatusOK, user)
+			var subOrganizationId *string
+			if user.SubOrganizationId.Valid {
```

Comment:
this shouldn't be possible, but our `api/register` endpoint isn't atomic and we create a database user before creating its sub-org and wallets. so for now, i think it makes sense to make this its own check  inside the user check. i think this should be fine for now though